### DISCUSSION TITLE
Give a climate card a slider option

### DISCUSSION THREAD

**Discussion by tomtom13** [home-assistant/frontend#22252] [Other feature requests]
Hi, it would be nice to be able to create a small tile for thermostat / climate which would have a slider just like a light entity does rather than +/- buttons alone. It would also be nice to add option to have + / - buttons on each side to the slider.
And it would be VERY VERY nice to allow slider also go UP / DOWN rather than just side to side.

**Comment by craigmcfly** [comment]
Yes! Please! I have a dashboard in portrait mode and a slider for the climate entity would be great.
